src/pspm_interpolate.m

```diff
@@ -79,21 +78,21 @@
 if nargin<1
   warning('ID:missing_data', 'No data.\n');
   return;
-end;
+end
 
 if isempty(indata)
   warning('ID:missing_data', 'Input data is empty, nothing to do.');
   return;
-end;
+end
 
 % set options ---
-try options.overwrite; catch, options.overwrite = 0; end;
-try options.method; catch, options.method = 'linear'; end;
-try options.channels; catch, options.channels = []; end;
-try options.newfile; catch, options.newfile = 0; end;
-try options.channel_action; catch, options.channel_action = 'add'; end;
-try options.extrapolate; catch, options.extrapolate = 0; end;
-try options.dont_ask_overwrite; catch, options.dont_ask_overwrite = 0; end;
+try options.overwrite; catch, options.overwrite = 0; end
+try options.method; catch, options.method = 'linear'; end
+try options.channels; catch, options.channels = []; end
+try options.newfile; catch, options.newfile = 0; end
+try options.channel_action; catch, options.channel_action = 'add'; end
+try options.extrapolate; catch, options.extrapolate = 0; end
+try options.dont_ask_overwrite; catch, options.dont_ask_overwrite = 0; end
```

Comment:
> Does it still need to be initialized if it's not used?
> 
> Ignore this message if it has other uses that the one you simplified in earlier in this push.

Thanks Jules for pointing out this. I have removed this line because the unused item should not be mentioned anymore.